src/main/java/appeng/client/render/tesr/InscriberTESR.java

```diff
@@ -203,7 +205,9 @@ private void renderItem(MatrixStack ms, final ItemStack stack, final float o, Ve
             // for direction=null, while a block-model will have their faces for
             // cull-faces, but not direction=null
             BakedModel model = itemRenderer.getModels().getModel(stack);
-            if (!model.getQuads(null, null, new Random()).isEmpty()) {
+            List<BakedQuad> quads = model.getQuads(null, null, new Random());
+            // Note: quads may be null for mods implementing FabricBakedModel without caring about getQuads.
+            if (quads != null && quads.isEmpty()) {
```

Comment:
```suggestion
            if (quads != null && !quads.isEmpty()) {
```

Shouldn't it be !quads.isEmpty() ?